xls/dslx/stdlib/float32.x

```diff
@@ -35,6 +35,9 @@ pub fn bias(unbiased_exponent_in: u9) -> u8 {
 }
 pub fn flatten(f: F32) -> u32 { apfloat::flatten<u32:8, u32:23>(f) }
 pub fn unflatten(f: u32) -> F32 { apfloat::unflatten<u32:8, u32:23>(f) }
+pub fn cast_from_fixed<NUM_SRC_BITS:u32>(s: sN[NUM_SRC_BITS]) -> F32 {
```

Comment:
Any chance you could toss this in the float64 and bfloat16 modules, too?

Reply:
Done.